api/src/modules/index.ts

```diff
@@ -1,4 +1,2 @@
 export * from './User';
-export * from './CoreUnit';
```

Comment:
Instead of moving a specific module into the package, please define a new `ExampleType` with actual example resolver (which for example can return a number of users in the database)

Reply:
see the adjustment